frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -516,13 +516,17 @@ private void prepareReleaseEdit(RenderRequest request, RenderResponse response)
                 if (isNullOrEmpty(id)) {
                     id = release.getComponentId();
                 }
+                component = client.getComponentById(id, user);
```

Comment:
Maybe do this before the if -block

Reply:
But the id might change inside this branch?

Reply:
You're right. Sorry, missed that.

However it sounds a little bit strange to me, that this id can be null.